domains/jacksonjperry03.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "jacksonjperry03",
+           "email": "jackson.perry@uri.edu",
+           "discord": "395990222833451008"
+        },
+    
+        "record": {
+            "CNAME": "https://jacksonjperry03.github.io/jackson-perry-portfolio/"
```

Comment:
Invalid CNAME record as it should be a hostname, that contains no protocols or URL paths. Also 404.